activejob/test/support/integration/adapters/sneakers.rb

```diff
@@ -84,7 +84,7 @@ def bunny_publisher
     end
 
     def bunny_queue
-      @queue ||= bunny_publisher.exchange.channel.queue "integration_tests", durable: true
+      @queue ||= bunny_publisher.send(:exchange).channel.queue "integration_tests", durable: true
```

Comment:
Hey guys, I can explore exposing the exchange back. Thanks for notifying me!
Will get back with updates soon